dev/tools/gen_date_localizations.dart

```diff
@@ -81,9 +81,11 @@ Future<void> main(List<String> rawArgs) async {
 // To regenerate run (omit --overwrite to print to console instead of the file):
 // dart --enable-asserts dev/tools/gen_date_localizations.dart --overwrite
 
-// ignore_for_file: public_member_api_docs
 '''
 );
+  buffer.writeln('''
+/// The subset of date symbols supported by the intl package supported by
```

Comment:
I think you need a connective word or two in these statements like: 

The subset of date symbols supported by the intl package which are supported by
The subset of date symbols supported by the intl package which are also supported by

Lint will be happy with any text at all of course :-)

Reply:
Thanks, clarified